### ISSUE TITLE
Add Italian as one of the main languages

### ISSUE THREAD

**Issue by RiccardoMasutti** [bisq-network/proposals#151]
## TL;DR
Add Italian language on the software, add Italian projects on Transifex (both website and subtitles).

### Motivation
- In Italy BTC is considered a currency
- Italian was the first translation of Bitcoin Core and has always been one of the largest communities in the world
- According to [Coinmap](https://coinmap.org/), 15.3% of the world’s stores that accept BTC payments are in Italy
- From a [Semrush's survey](https://finanza.lastampa.it/News/2019/10/28/e-commerce-in-italia-crescono-acquisti-e-fiducia-nei-metodi-pagamento-online/MTE1XzIwMTktMTAtMjhfVExC), in Italy, Bitcoin is the third payment method after Paypal and PostePay
- There are 100+ ATMs to directly withdraw virtual currencies and an active Localbitcoins community.

Unfortunately, Bisq is little known in Italy. I am already translating a large part of the software on my own and would like to see it online in the next few months, along with the website and the subtitles.
I just launched a Keybase channel, a Twitter account and a Telegram group and response was good so far.
I'm a well known Bitcoin speaker and teacher here in Italy and I always promote Bisq in my talks. The English language is often a difficult barrier to overcome for Italians. If Bisq were translated, I'm sure the use would be much simpler and more understandable.

**Comment by chimp1984** [comment]
Can you estimate costs for translating the Bisq app into Italian? I assume you would also like to have Italian for the webpage, so costs for that would be good as well.

I basically agree that it would help the Italian market to grow, but we also need to keep our expensed in balance with the potential growth potential. As trade volume and with that burned BSQ is not doing great at the moment being a bit more conservative with expenses might be prudent.

**Comment by MwithM** [comment]
Can we somehow estimate how the current translations are working, what are the results in terms of growth?
We're facing a chicken-egg dilemma here, without translations maybe there won't be ennough italians to growth, but without growth, we cannot translate.
If we had a reliable metric of growth due to translations, we could delay BSQ compensations in a way where "X downloads-visits equals Y BSQ", until a top of BSQ/word rate.
With a system like this, we could fund development that don't have a direct and instant impact impact on growth and which could be a failure, like marketing and translations.

**Comment by RiccardoMasutti** [comment]
> Can we somehow estimate how the current translations are working, what are the results in terms of growth?
> We're facing a chicken-egg dilemma here, without translations maybe there won't be ennough italians to growth, but without growth, we cannot translate.
> If we had a reliable metric of growth due to translations, we could delay BSQ compensations in a way where "X downloads-visits equals Y BSQ", until a top of BSQ/word rate.
> With a system like this, we could delay funding initiatives that don't have a direct and instant impact impact on growth and which could be a failure, like marketing and translations.

**Comment by RiccardoMasutti** [comment]
> Can you estimate costs for translating the Bisq app into Italian? I assume you would also like to have Italian for the webpage, so costs for that would be good as well.
> 
> I basically agree that it would help the Italian market to grow, but we also need to keep our expensed in balance with the potential growth potential. As trade volume and with that burned BSQ is not doing great at the moment being a bit more conservative with expenses might be prudent.

Software 18953 words * 0.075 = 1421 USD
Can't check website and subtitles, but should be ¹/3

**Comment by erciccione** [comment]
> From a Semrush's survey, in Italy, Bitcoin is the third payment method after Paypal and PostePay

@RiccardoMasutti Do you have a direct link to this survery? Every article i find about it link to this article of LaStampa, which doesn't offer any source and i cannot find the actual survey anywhere. I ask because, as an Italian, it's hard to believe it.

About the proposal itself, i agree. English is generally a barrier for italians and i think most of them won't use Bisq until it will be available in Italian.

**Comment by RiccardoMasutti** [comment]
> > From a Semrush's survey, in Italy, Bitcoin is the third payment method after Paypal and PostePay
> 
> @RiccardoMasutti Do you have a direct link to this survery? Every article i find about it link to this article of LaStampa, which doesn't offer any source and i cannot find the actual survey anywhere. I ask because, as an Italian, it's hard to believe it.
> 
> About the proposal itself, i agree. English is generally a barrier for italians and i think most of them won't use Bisq until it will be available in Italian.

LaStampa is one of the oldest and most reputable newspaper in Italy. The same was reported by [Yahoo Finance](https://finance.yahoo.com/news/italians-prefer-shop-online-bitcoin-214058647.html), [Decrypt](https://decrypt.co/10910/italians-prefer-shop-online-with-bitcoin-over-visa-mastercard-american-express-report), [TheNextWeb](https://thenextweb.com/hardfork/2019/10/31/italians-prefer-bitcoin-visa-mastercard-online-payments/). Just to name a few

**Comment by erciccione** [comment]
It is reputable as long as they provide sources for their claims, especially if we consider that Italian press [cannot be considered reliable](https://rsf.org/en/italy). The other articles you linked to seem to be copy-pasted and they all refer to the article from LaStampa as source. Anyway, i don't want to go too off topic :)

**Comment by chimp1984** [comment]
> > Can you estimate costs for translating the Bisq app into Italian? I assume you would also like to have Italian for the webpage, so costs for that would be good as well.
> > I basically agree that it would help the Italian market to grow, but we also need to keep our expensed in balance with the potential growth potential. As trade volume and with that burned BSQ is not doing great at the moment being a bit more conservative with expenses might be prudent.
> 
> Software 18953 words * 0.075 = 1421 USD
> Can't check website and subtitles, but should be ¹/3

This sounds like a reasonable investment for me. So I would support that.

**Comment by SimoneBovi** [comment]
As an Italian myself too, I totally agree with the purposes of this proposal.

English is a huge barrier here in Italy but we have a huge Bitcoin community as well :)

**Comment by ripcurlx** [comment]
> Can you estimate costs for translating the Bisq app into Italian? I assume you would also like to have Italian for the webpage, so costs for that would be good as well.
> 
> I basically agree that it would help the Italian market to grow, but we also need to keep our expensed in balance with the potential growth potential. As trade volume and with that burned BSQ is not doing great at the moment being a bit more conservative with expenses might be prudent.

I think one way to estimate potential for new core languages could be to find consensus first for translating the most important pages of the website and afterwards if we see that traffic increases through this we can talk about adding it to the client as well (which is where most of the translation work is done).  The reasoning for adding Italian makes sense to me as I have similar experience on the need for translation in Italian in the gaming sector.

**Comment by arunasurya** [comment]
@RiccardoMasutti , thank you for writing this proposal! It has helped me clarify better guidelines for writing language proposals. I have added them [here](https://github.com/arunasurya/bisqtransifex/blob/master/translationguidelines.md), take a look if you like. I like the suggestion by @ripcurlx. Perhaps, translators can start by translating Bisq Vision page which should not take much time, and then we can determine how useful it is...

**Comment by mpolavieja** [comment]
Approved by DAO voting cycle 8
